### DISCUSSION TITLE
Error: ntfs-3g has been disabled because it requires FUSE!

### DISCUSSION THREAD

**Discussion by Exanys** [Homebrew/discussions#1745] [Casks]
Hello, 
I'm trying to install ntfs-3g, but getting an error that it requires FUSE.
The problem is I have Fuse already installed on my Mac Air M1 and in the boot menu I have already enable applications out of Apple.
Do you have any solution?
`brew install ntfs-3g                   
Error: ntfs-3g has been disabled because it requires FUSE!`

**Comment by gromgit** [comment]
All FUSE formulae have been removed from Homebrew core for reasons: https://github.com/Homebrew/homebrew-core/pull/64491

I'm currently maintaining all the FUSE formulae (that can still be built from source) in [my own dedicated tap](https://github.com/gromgit/homebrew-fuse). All formulae names are suffixed with `-mac` to avoid name conflicts with the core tap, so:
```
brew install gromgit/dev/ntfs-3g-mac
```
should take care of your needs, but do read the [README](https://github.com/gromgit/homebrew-fuse/blob/main/README.md) to understand certain things I've done.

**Reply by Exanys** [reply]
So if I understand, I need to have macfuse v2.x.x?
Or which FUSE should I install?

**Reply by Exanys** [reply]
I have macFuse v 4.1.2 and i go this error:
`brew install ntfs-3g-mac
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> Updated Formulae
Updated 3 formulae.

==> Installing ntfs-3g-mac from gromgit/fuse
==> Downloading https://raw.githubusercontent.com/Homebrew/formula-patches/3933b
######################################################################## 100.0%
==> Downloading https://tuxera.com/opensource/ntfs-3g_ntfsprogs-2017.3.23.tgz
==> Downloading from https://download.tuxera.com/opensource/ntfs-3g_ntfsprogs-20
######################################################################## 100.0%
==> Patching
==> Applying lowntfs-3g.c.patch
patching file src/lowntfs-3g.c
Hunk #1 succeeded at 1927 (offset -62 lines).
==> ./configure --prefix=/opt/homebrew/Cellar/ntfs-3g-mac/2017.3.23 --exec-prefi
==> make
Last 15 lines from /Users/vojta/Library/Logs/Homebrew/ntfs-3g-mac/02.make:
clang -DHAVE_CONFIG_H -I. -I..     -I../include/ntfs-3g -g -O2 -Wall -c -o ntfs_3g_probe-ntfs-3g.probe.o `test -f 'ntfs-3g.probe.c' || echo './'`ntfs-3g.probe.c
clang -DHAVE_CONFIG_H -I. -I..     -DFUSE_USE_VERSION=26 -D_FILE_OFFSET_BITS=64 -I/usr/local/include/fuse -I../include/ntfs-3g -DPLUGIN_DIR=\"/opt/homebrew/Cellar/ntfs-3g-mac/2017.3.23/lib/ntfs-3g\" -g -O2 -Wall -c -o ntfs_3g-ntfs-3g.o `test -f 'ntfs-3g.c' || echo './'`ntfs-3g.c
clang -DHAVE_CONFIG_H -I. -I..     -DFUSE_USE_VERSION=26 -D_FILE_OFFSET_BITS=64 -I/usr/local/include/fuse -I../include/ntfs-3g -DPLUGIN_DIR=\"/opt/homebrew/Cellar/ntfs-3g-mac/2017.3.23/lib/ntfs-3g\" -g -O2 -Wall -c -o ntfs_3g-ntfs-3g_common.o `test -f 'ntfs-3g_common.c' || echo './'`ntfs-3g_common.c
clang -DHAVE_CONFIG_H -I. -I..     -DFUSE_USE_VERSION=26 -D_FILE_OFFSET_BITS=64 -I/usr/local/include/fuse -I../include/ntfs-3g -DPLUGIN_DIR=\"/opt/homebrew/Cellar/ntfs-3g-mac/2017.3.23/lib/ntfs-3g\" -g -O2 -Wall -c -o lowntfs_3g-lowntfs-3g.o `test -f 'lowntfs-3g.c' || echo './'`lowntfs-3g.c
clang -DHAVE_CONFIG_H -I. -I..     -DFUSE_USE_VERSION=26 -D_FILE_OFFSET_BITS=64 -I/usr/local/include/fuse -I../include/ntfs-3g -DPLUGIN_DIR=\"/opt/homebrew/Cellar/ntfs-3g-mac/2017.3.23/lib/ntfs-3g\" -g -O2 -Wall -c -o lowntfs_3g-ntfs-3g_common.o `test -f 'ntfs-3g_common.c' || echo './'`ntfs-3g_common.c
lowntfs-3g.c:31:10: fatal error: 'fuse_lowlevel.h' file not found
#include <fuse_lowlevel.h>
         ^~~~~~~~~~~~~~~~~
/bin/sh ../libtool  --tag=CC   --mode=link clang  -I../include/ntfs-3g -g -O2 -Wall  -lintl -framework CoreFoundation -o ntfs-3g.probe ntfs_3g_probe-ntfs-3g.probe.o ../libntfs-3g/libntfs-3g.la -lc
1 error generated.
make[2]: *** [lowntfs_3g-lowntfs-3g.o] Error 1
make[2]: *** Waiting for unfinished jobs....
libtool: link: clang -I../include/ntfs-3g -g -O2 -Wall -o .libs/ntfs-3g.probe ntfs_3g_probe-ntfs-3g.probe.o  ../libntfs-3g/.libs/libntfs-3g.dylib -lintl -lc -framework CoreFoundation
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2

Do not report this issue to Homebrew/brew or Homebrew/core!`

**Reply by gromgit** [reply]
Sorry, I don't have an M1 machine, so will need your help to figure out how to fix this. As this concerns a private tap, please [open an issue on my repo](https://github.com/gromgit/homebrew-fuse/issues/new/choose) and we'll take it from there. Thanks much!
